src/layouts/home/user-list-item/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import { ChatBriefViewResponse } from '@api/index';
+import { getDateDistance } from '@layouts/home/user-list-item/utils';
+import ChatAvatarSvg from '@public/icons/chat-avatar.svg';
+import GroupAvatarSvg from '@public/icons/group-avatar.svg';
+
+const userId = 'user001'; //temporary variable
+const unreadableMessages = 32; //temporary variable
+
+interface Props {
+  item: ChatBriefViewResponse;
+  setContentIsVisible: Dispatch<SetStateAction<boolean>>;
+}
+
+const UserListItem: React.FC<Props> = ({ item, setContentIsVisible }) => {
+  const { chatName, participants, lastMessage } = item;
+  const avatarStyles = 'mr-[10px] w-10 h-10 min-w-[40px] min-h-[40px]';
+
+  const title = chatName
+    ? chatName
+    : participants
+        ?.flatMap((item) => (item.id === userId ? [] : item.username))
+        .join(', ');
+
+  const time = lastMessage?.timestamp
+    ? getDateDistance(lastMessage?.timestamp)
+    : 'Recently';
+
+  return (
+    <li
```

Comment:
It is better to use `li` in the place where you iterate over the list, because it is an independent element, and it can be reused in other places